crates/gitbutler-testsupport/src/lib.rs

```diff
@@ -289,3 +286,9 @@ pub(crate) static DRIVER: Lazy<PathBuf> = Lazy::new(|| {
 /// Note that this can't be used if secrets themselves are under test as it' doesn't act
 /// like a real store, i.e. stored secrets can't be retrieved anymore.
 pub mod secrets;
+
+pub fn set_test_data_dir<P: AsRef<std::path::Path>>(path: P) {
+    unsafe {
+        std::env::set_var("TEST_APP_DATA_DIR", path.as_ref());
```

Comment:
Not sure about this - it's set to a temp path, which is usually done per test, so tests would interfere with each other now.